blog/_src/posts/2017-08-11-on-being-a-pc-member-rebuttals-and-responses.md

```diff
@@ -0,0 +1,104 @@
+    Title: Reviews and author responses: we should stop asking for short responses
+    Date: 2017-08-11T14:29:41
+    Tags: by Gabriel Scherer
+
+This year I reviewed many ICFP submissions, and got to be on the
+receiving end of equally many author responses (also sometimes called,
+somewhat combatively, rebuttals). I found that there was a large
+difference between the official written advice on author responses and
+what I, as a reviewer reading the responses, found effective. In
+particular, I now believe that limiting yourself to 500 words should
+strongly be avoided -- we should even stop giving that advice.
+
+<!-- more -->
+
+This year, I had the honor (and accompanying load of work) of being
+a Program Committee (PC) member at the ICFP conference. It was first
+time being a PC member at a conference, and I found it extremely
+pleasant and interesting, thanks to the authors who sent us articles
+to review, my fellow PC members, and the ever-smiling careful
+balancing work of our PC chair, Mark Jones. It was also a lot of work,
+starting with 18 reviews to do over a month and a half, an intense PC
+meeting, and the new "second phase" process with the opportunity for
+authors and reviewers to exchange feedback on changes requested by the
+program committee.
+
+There is little guidance on how to write author responses, although
+this [blog
+post](http://www.pl-enthusiast.net/2014/09/17/advice-writing-author-response/)
+by Michael Hicks on pl-enthusiast is quite good. One thing that is
+obvious as a reviewer and is only slightly brushed in this post,
+however, is that author responses should *not* aim to fit a 500 words
+limit, and in fact I believe that it is a very bad idea to do so.
+
+As for most conference, the ICFP review system recommends (in writing)
+to authors to keep their response to 500 words (some systems also
+highlight words after those in red to make the point clear). Don't do
+this! The least convincing responses I have seen are those that
+followed this recommendation.
+
+We have a frustrating situation where the explicit rule is different
+from the thing people do in practice. This is bad for newcomers that
+do not know the norms and cannot tell if ignoring the rule may hurt
+them. This is the point of this blog post:
+
+- If you are an author, please know that disrespecting the 500-words
+  limit is the *right thing* to do. You should also know, of course,
+  that people have limited time, so keep the main body of your
+  response reasonably long. (I spent a day and a half on your paper
+  already, I am willing to spend 10 additional minutes reading the
+  response.)
+
+- If you are a program committee chair or a Magical HotCRP Wizard,
+  please remove this silly recommendation to keep responses to 500
+  words.
```

Comment:
~Can you offer something to replace it with? "Keep your responses short and to the point."~

Put a pointer here to the suggestion at the bottom